test-utils/TestApplications.scala

```diff
@@ -105,7 +105,7 @@ trait TestApplications {
 
     final def withDescription(description: Option[String]): Application = app.copy(description = description)
 
-    final def withTeamMember(email: String, userRole: Role): Application = app.copy(collaborators = app.collaborators + Collaborator(email, userRole, Some(UserId.random)))
+    final def withTeamMember(email: String, userRole: Role): Application = app.copy(collaborators = app.collaborators + Collaborator(email, userRole, UserId.random))
```

Comment:
should this random id be retrieved from idOf ?

Reply:
Yes